content/quiz_events.md

```diff
@@ -4,21 +4,134 @@
+
+### Deity Rule
+- The correct person of the deity must be given. 
+- The quizzer may not move between persons of the deity when answering. Giving the name of a deity that is not correct is considered giving incorrect information.
+- If a quizzer refers to any person of the deity as either “God” or “Lord”, these must be clarified by the quizzer within 30 seconds when the text requires a more specific answer. 
+- “Jesus” and “Christ” are Interchangeable (but refer to the person of the Son) and the quizzer will be called correct.
+- If a pronoun is used in the text to name the deity, any of the names for that person of the deity within context may be considered an acceptable answer when clarifying.
+    - If the pronoun naming the deity is identified by the quizzer using a name that correctly identifies the deity, but with an identification that is out of context, the quizzer will be considered to have not stayed in context.
+- When there is a Special Name of the deity given in the text, the Special Name must be given in the quizzer’s answer.  (i.e.  “The Lord God Almighty”, “The Bread of Life”, “Spirit of the Living God”.)
+    - If the quizzer states any other Special Names which are not in context, the quizzer will be considered to have not stayed in context, even if they refer to the same person of the deity. Example: Jesus refers to himself as “the bread of life” in John 6 and “the good shepherd” in John 10. 
+- When a quizzer gives other names for the same person of the deity, used within context, these should be allowed to be clarified within 30 seconds when a more specific answer is needed.
+
 ### Correct
+The quizzer's response is correct when:
+- It fulfills all requirements specific to the Question Type.
+- The quizzer stays in Context.
+- It contains the information requested.
+    - If the quizmaster did not complete the reading of the question, the quizzer's answer must include the information in the remainder of the question as well as the answer.
+    - It is not required to be in question and answer form, except in the case of a reference question.
+    - The quizzer’s answer must include all unique words included in both the question and the answer.
+
+Mispronounced names are still recognizable as the answer.
+
+Only the first response of the quizzer will be considered, however, if the response by the quizzer is in context and is not incorrect, the quizzer will be allowed their full 30 seconds to give the additional information necessary to satisfy the question and the answer.  Example: INT Who has been making it grow? A. God.  If the quizzer said “Apollos” they have given incorrect information (even though the name “Apollos” is in context) and would be called incorrect.  If they had said “I planted the seed”, “Apollos watered it”, they have not given any incorrect information, are in context and will be given the 30 seconds to supply the correct answer required. 
```

Comment:
Slightly reworded. I think I maintained functional parity.

Reply:
Examples should be contained under example header. But otherwise, w00t.